<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icon (1).png</normaloff>icon (1).png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="ffmpegPath">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>20</y>
      <width>431</width>
      <height>45</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_1">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>20</y>
      <width>96</width>
      <height>45</height>
     </rect>
    </property>
    <property name="text">
     <string>选择ffmpeg程序：</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="BrowseButton_1">
    <property name="geometry">
     <rect>
      <x>615</x>
      <y>19</y>
      <width>75</width>
      <height>45</height>
     </rect>
    </property>
    <property name="text">
     <string>浏览</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="OutPutPath">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>120</y>
      <width>431</width>
      <height>45</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="BrowseButton_3">
    <property name="geometry">
     <rect>
      <x>615</x>
      <y>119</y>
      <width>75</width>
      <height>45</height>
     </rect>
    </property>
    <property name="text">
     <string>浏览</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>120</y>
      <width>101</width>
      <height>45</height>
     </rect>
    </property>
    <property name="text">
     <string>选择文件夹路径：</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="JPGButton">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>210</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>导出为JPG文件</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PNGButton">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>210</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>导出为PNG文件</string>
    </property>
   </widget>
   <widget class="QPushButton" name="JPG2MP4">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>210</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>JPG文件合并为MP4</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="VideoPath">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>70</y>
      <width>431</width>
      <height>45</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="BrowseButton_2">
    <property name="geometry">
     <rect>
      <x>615</x>
      <y>69</y>
      <width>75</width>
      <height>45</height>
     </rect>
    </property>
    <property name="text">
     <string>浏览</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>70</y>
      <width>84</width>
      <height>45</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="text">
     <string>选择视频文件：</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
